packages/web-react/src/components/Toast/demo/ToastDynamicToastQueue.tsx

```diff
@@ -30,10 +31,10 @@ const ToastDynamicToastQueue = () => {
       isOpen: true,
       message: (
         <>
-          I was first!{' '}
-          <Link href="#" color="inverted" isUnderlined>
+          <ToastBarMessage>I was first!</ToastBarMessage>
+          <ToastBarLink href="#" color="inverted" isUnderlined>
```

Comment:
It is necessary to set a color for this subcomponent? I think that the color should be determined by the component, e.g. we are setting color only in the uppermost component and the inner ones should be working just fine.